.github/workflows/test-report.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Test Report'
+on:
+  workflow_run:
+    workflows: ['tests']                     # runs after CI workflow
+    types:
+      - completed
+jobs:
+  report:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: dorny/test-reporter@v1
+      with:
+        artifact: verification-results
+        name: Verification Tests
```

Comment:
Why are we using the name "test" here?  Isn't it reporting verification metrics?  Or is it related to Dafny's `:test` somehow?

Reply:
Nope, I just did a poor job of editing the template. :) I'm changing it to "Verification Results"